DEVELOPMENT.md

```diff
@@ -0,0 +1,92 @@
+# Development
+This file highlight the installation process of getting started with Ersilia, Ersilia Artifacts and Creating a PR.
+
+## Getting Started
+Ersilia can be run either on local computer or with the use of GitHub codespace. On your local computer, ersilia is designed to run specific environment
```

Comment:
We can rephrase "On your local computer" to, "Ersilia is designed to run on Linux or Linux-like environments and at present is not supported on Windows systems".